CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Fix
+
+* Flaky JsonSerializerTest has been fixed (#2021)
```

Comment:
```suggestion
```

We don't need that, we only add user-facing changes to the changelog.

Reply:
Oops, wasn't thinking this through. Thanks!